src/css/Splits.scss

```diff
@@ -40,7 +46,9 @@ $split-icon-two-row-size: $two-row-height - $split-column-padding;
       }
     }
 
-    @include time;
+    &.could-show-separator {
```

Comment:
idk this still feels like a weird hack to me. While it doesn't "flicker" anymore, it still means the last split's icon is smaller than the rest.

Reply:
In order to prevent the flickering, we need to compromise in at least one of the cases (either when showing the thick separator or when showing the thin separator) because the amount of size that the split gets is not consistent. In the original LiveSplit, we solve this by making the last split 1 pixel taller and adding a blank pixel row below the thin separator. Let me know if you have a suggestion for a better solution, but there won't be any "clean" solution because it's not possible to maintain both a consistent size and a consistent position when the amount of blank space available is changing.

Reply:
That's why the thick separator is just an "overlay" in the native renderer so it doesn't affect anything.

https://github.com/LiveSplit/livesplit-core/blob/master/src/rendering/component/splits.rs#L156-L169

Reply:
I tried doing that and it didn't look good: https://github.com/LiveSplit/LiveSplitOne/pull/244#issuecomment-582683352

Reply:
I'm not sure I see what doesn't look good about it?!

Reply:
Maybe it would be better if the border was extended up instead of down though? idk

Reply:
Yes it should be equally growing up as it does down compared to the thin separator it would be

Reply:
The icon is getting cut off:

![Screenshot_20200207-184010_Chrome](https://user-images.githubusercontent.com/8262173/74075678-4bffaf00-49d9-11ea-8163-0701aaa9a70b.jpg)

Reply:
Yes then it's probably either positioned too far down or the icons are just too big.

Reply:
I'll move the top borders up by a pixel and get rid of the weird padding fix.